aldryn_cms/urls.py

```diff
@@ -3,8 +3,12 @@
 
 from cms.sitemaps import CMSSitemap
 
+from .views import check_uninstall_ok
+
 
 urlpatterns = [
+    # Not sure if we should hardcode admin path like this
+    url(r'^admin/~cmscloud-api/check-uninstall/$', check_uninstall_ok),
```

Comment:
Is that the same URL which is already user? I would prefer to have an aldryn specific prefix.
Is that resource protected in any way?